src/polyswarmd/__init__.py

```diff
@@ -134,7 +134,7 @@ def before_request():
     # Ignore prefix if present
     try:
         api_key = request.headers.get('Authorization').split()[-1]
-    except:
+    except IndexError:
```

Comment:
This will fail if no auth header is present, for checking auth i think an `except Exception` is warranted and that seems to satisfy code climate